scripts/flow/v11/flowPatcher.js

```diff
@@ -0,0 +1,102 @@
+import {FlowInfo, getTokenByIdOrActorId, processFlowInfo} from "../common.js";
+import {isV10} from "../../utils.js";
+
+const _patchFlow = (flowId, patch) => {
+	const originalFlow = game.lancer.flowSteps.get(flowId);
+	if (!originalFlow) throw new Error(`Could not find flow "${flowId}" to patch!`);
+
+	game.lancer.flowSteps.set(flowId, async (...args) => {
+		await patch(...args);
+		return originalFlow(...args);
+	})
+};
+
+/**
+ * @param state
+ * @param {?string} fallbackActionIdentifier Identifier used if the flow does not specify one.
+ * @return {FlowInfo}
+ */
+const _getFlowInfo = (state, {fallbackActionIdentifier = null} = {}) => {
+	const zippedTargetInfo = Array.from(
+		{
+			length: Math.max(
+				state.data.acc_diff?.targets?.length || 0,
+				state.data.hit_results?.length || 0,
+			),
+		},
+		(_, i) => ({
+			target: state.data.acc_diff?.targets?.[i],
+			hit_result: state.data.hit_results?.[i],
+		}),
+	);
+
+	return new FlowInfo({
+		sourceToken: getTokenByIdOrActorId(state.actor.token?.id || state.actor?.id),
+		actionIdentifier: state.item?.system?.lid || fallbackActionIdentifier,
+		targetTokens: zippedTargetInfo
+			.map(({target}) => target.target)
+			.filter(Boolean),
+		targetsMissed: new Set(
+			zippedTargetInfo
+				.filter(({hit_result}) => !hit_result?.hit)
+				.map(({target}) => target?.target?.id)
+				.filter(Boolean)
+		),
+	});
+}
+
+const _flowPatch_printAttackCard = async (state) => {
+	if (!state.data) return;
+
+	const flowInfo = _getFlowInfo(
+		state,
+		{
+			fallbackActionIdentifier: state.data.attack_type === "Melee"
+				? "lwfx_default_melee"
+				: "lwfx_default_ranged"
+		},
+	);
+	if (flowInfo == null) return;
+
+	await processFlowInfo(flowInfo);
+};
+
+const _flowPatch_printTechAttackCard = async (state) => {
+	if (!state.data) return;
+
+	const flowInfo = _getFlowInfo(state, {fallbackActionIdentifier: "default_tech_attack"});
+	if (flowInfo == null) return;
+
+	await processFlowInfo(flowInfo);
+};
+
+const _flowPatch_printStabilizeResult = async (state) => {
+	if (!state.data) return;
+
+	const flowInfo = _getFlowInfo(state, {fallbackActionIdentifier: "lwfx_stabilize"});
+	if (flowInfo == null) return;
+
+	await processFlowInfo(flowInfo);
+};
+
+const _flowPatch_printActionUseCard = async (state) => {
+	if (!state.data) return;
+
+	const flowInfo = _getFlowInfo(state);
+	if (flowInfo == null) return;
+
+	await processFlowInfo(flowInfo);
+};
+
+const _onReady = () => {
+	if (isV10()) return;
+
+	_patchFlow("printAttackCard", _flowPatch_printAttackCard);
+	_patchFlow("printTechAttackCard", _flowPatch_printTechAttackCard);
+	_patchFlow("printStabilizeResult", _flowPatch_printStabilizeResult);
+	_patchFlow("printActionUseCard", _flowPatch_printActionUseCard);
```

Comment:
The "print" part of a flow posts it to chat. We wrap these to mimic the v10 version.

It is possible to wrap earlier parts of the flow, but "print" is good because this is when the final result is presented to the user -- usually when we want to play an effect!